modules/using-wildcard-routes.adoc

```diff
@@ -0,0 +1,286 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-controller
+
+[id="using-wildcard-routes_{context}"]
+= Using wildcard routes
+
+The HAProxy Ingress Controller has support for wildcard routes. The Ingress Operator uses `wildcardPolicy` to configure the `ROUTER_ALLOW_WILDCARD_ROUTES` environment variable of the Ingress Controller.
+
+The default behavior of the Ingress Controller is to admit routes with a wildcard policy of `None`, which is backwards compatible with existing `IngressController` resources.
+
+.Procedure
+
+. Configure the wildcard policy.
+.. Use the following command to edit the `IngressController` resource.
++
+----
+$ oc edit IngressController
+----
++
+.. Under `spec`, set the `wildcardPolicy` field to `WildcardsDisallowed` or `WildcardsAllowed`:
++
+[source,yaml]
+----
+spec:
+  routeAdmission:
+    wildcardPolicy: WildcardsDisallowed # or WildcardsAllowed
+----
+
+
+.Samples for using a secure wildcard edge terminated route
+
+This example reflects TLS termination occurring on the Ingress Controller before traffic is proxied to the destination. Traffic sent to any hosts in the subdomain
+`example.org` (`*.example.org`) is proxied to the exposed service.
```

Comment:
Seems the following example has the domain `*.example.test` instead of `example.org`.